### PR TITLE
Add research doc for one-round parallel queuer

### PR DESCRIPTION
**Pull request by CLEckhardt**
Adds a doc to community/planning that examines the feasibility and practicality of one-round parallel queuing strategies.

### CODE REVIEW

File: community/planning/queuer_strat_one_round_parallel.md [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+# One-round parallel queuer strategy
+
+## Overview
+
+Below we examine the practicality and feasibility of queuing parallelization
+strategies. The goal of such strategies is to increase the average
+througput of a queuing strategy while maintaining a batch submission order
```
Comment (shannynalayna) [inline community/planning/queuer_strat_one_round_parallel.md:12]: sp/througput/throughput

File: community/planning/queuer_strat_one_round_parallel.md [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+guarantee (which ensures data integrity). Since we can't directly assess the
+average througput of a strategy at this point in Grid's development, we will
```
Comment (shannynalayna) [inline community/planning/queuer_strat_one_round_parallel.md:14]: Same sp as above

**Review by ameliabradley — CHANGES_REQUESTED** [review]
Should there be a link to this new document?

File: community/planning/queuer_strat_one_round_parallel.md (hunk 1 above)
Comment (CLEckhardt) [reply community/planning/queuer_strat_one_round_parallel.md]: fixed, thanks!

File: community/planning/queuer_strat_one_round_parallel.md (hunk 2 above)
Comment (CLEckhardt) [reply community/planning/queuer_strat_one_round_parallel.md]: fixed, thanks!

**Review by ameliabradley — APPROVED** [review]

**Review by shannynalayna — APPROVED** [review]

**Pull request merged**